docs/simulation/functionalities/simulate-sample-heterogeneity.md

```diff
@@ -3,8 +3,110 @@ layout: default
-nav_order: 5
+nav_order: 3
 ---
 
 # Simulate sample heterogeneity
-{: .no_toc }
\ No newline at end of file
+{: .no_toc }
+
+Sample heterogeneity is characterized by the presence of molecule sub-populations with different characteristics.
+It can be simulated for various characteristics by using one of the following methods:
+
+1. TOC
+{:toc}
+
+
+---
+
+## Parameter deviations
+
+This method can be used to introduce sample heterogeneity in FRET states 
+[*FRET*<sub>*j*</sub>](){: .math_var }, donor emission in absence of acceptor 
+[*I*<sub>tot,em</sub>](){: .math_var } and gamma factors 
+[*&#947;*](){: .math_var }.
+
+Parameters are randomly generated for each molecule by drawing from Gaussian distributions with means respectively defined in fields 
+`FRETj`, `Itot,em`, `γ`, and standard deviation in fields `wFRETj`, `wItot,em` and `wγ`.
+
+This method produces a continuous sample heterogeneity, *i.e.*, a broadening of molecule characteristics as opposed to the creation of sub-populations of molecules with distinguishable characteristics.
+
+
+---
+
+## Molecule-specific pre-sets
+
+This method can be used to introduce sample heterogeneity in FRET states 
+[*FRET*<sub>*j*</sub>](){: .math_var }, state transition rates 
+[*k*<sub>*jj'*</sub>](){: .math_var }, donor emission in absence of acceptor 
+[*I*<sub>tot,em</sub>](){: .math_var }, gamma factors 
+[*&#947;*](){: .math_var } and PSF widths 
+[*w*<sub>det</sub>](){: .math_var }.
+
+Parameters are defined individually for each molecule by loading a pre-set parameter file with the respective variables `FRET`, `trans_rates`, `tot_intensity`, `gamma` and `psf_width` set accordingly. 
+For more information about the structure and use of pre-set parameter file, please refer to 
+[Pre-set parameters](../panels/panel-molecules.html#pre-set-parameters).
+
+This method allows to custom the type of sample heterogeneity by either producing a broadening of molecule characteristics or by creating sub-populations of molecules with distinguishable characteristics.
+
+As an example, this script creates two distinct molecule sub-populations of equal size, with one population allowing step-wise and reversible state transitions, and one getting trapped after an off-path state transition:
+
+```matlab
+%%%%% EDIT HERE %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% defines 4-by-4 transition rates for the first population
+trans_rates1 = [ 0   0.1 0   0
+		 0.1 0   0.1 0
+		 0   0.1 0   0.1
+		 0   0   0.1 0];
+		
+% defines 4-by-4 transition rates for the second population
+trans_rates2 = [ 0   0.1 0   0
+		 0.1 0   0   0.1
+		 0   0   0   0
+		 0   0   0   0];
+
+% replicate the first and second transition rates matrix for half of the molecules with the repmat function
+trans_rates1 = repmat(trans_rates1,[1 1 N/2]);
+trans_rates2 = repmat(trans_rates2,[1 1 N/2]);
+
+% concatenate both sub-populations into one sample
+trans_rates = cat(3,trans_rates1,trans_rates2);
+
```

Comment:
3rd example for state heterogeneity: just simulate different rates with the very same FRET efficiency. Because, state degeneracy can introcude kinetic heterogeneity.

Reply:
The alternative method that you describe is the second method: use presets.
Anyway, for introducing sample heterogeneity or not, such partitioned transition rate matrix must work don't you think?

Reply:
The thrid method was removed from the section after we discussed the problem.